deploy/Chart/templates/ucp/deployment.yaml

```diff
@@ -63,6 +63,10 @@ spec:
         volumeMounts:
         - name: config-volume
           mountPath: /etc/config
+        {{- if eq .Values.global.aws.irsa.enabled true }}
+        - name: aws-iam-token
```

Comment:
here and on line 85, aws-irsa-token

Reply:
we are not changing it, so keeping it as is